tools/ubergen/bin/ubergen.ts

```diff
@@ -424,7 +424,6 @@ const IGNORED_FILE_NAMES = new Set([
   '.gitignore',
   '.jest.config.js',
   '.jsii',
-  '.npmignore',
```

Comment:
Interesting! I didn't know that `.npmignore` can be placed in subdirectories of a package and be respected by "npm pack"...

Reply:
It does indeed.

https://docs.npmjs.com/cli/v7/using-npm/developers#keeping-files-out-of-your-package

> Like `git`, `npm` looks for `.npmignore` and `.gitignore` files in all subdirectories of your package, not only the root directory.